### ISSUE TITLE
Files are created in weird locations (for Linux)

### ISSUE THREAD

**Issue by christofmuc** [christofmuc/KnobKraft-orm#320]
Again from @fractalf, the observation is:

Files on /home/user

After running the binary I see these folders created

~/.config/KnobKraft
~/KnobKraft-Adaptations
~/KnobKraftOrm

The first is ofc totally fine, but creating those other folders on the root of the home folder is pretty bad practice on a Linux system. You should consider moving them to ~/Documents/KnobKraft or something like that.

**Comment by christofmuc** [comment]
My comment was:

Yes, the files are very Unlinuxy. The code from JUCE is trying to be portable. let's have a look:

    /** The folder in which applications store their persistent user-specific settings.
        On Windows, this might be "\Documents and Settings\username\Application Data".
        On the Mac, it might be "~/Library". If you're going to store your settings in here,
        always create your own sub-folder to put them in, to avoid making a mess.
        On GNU/Linux it is "~/.config".
    */
    userApplicationDataDirectory,

    /** A place to put documents which are shared by all users of the machine.
        On Windows this may be somewhere like "C:\Users\Public\Documents", on OSX it
        will be something like "/Users/Shared". Other OSes may have no such concept
        though, so be careful.
    */
    commonDocumentsDirectory,

So the Linux ~/Documents is not implemented by the Library.
That's probably why I used the userHomeDirectory instead.

I am not really sure why it is creating a ~/KnobKraftOrm directory. What's in there?

**Comment by firstnevyn** [comment]
```
nevyn@edentata:~$ ls -l ~/KnobKraftOrm/
total 4
-rw-r--r-- 1 nevyn nevyn 1800 Jun 17 19:10 KnobKraftOrm.settings
```

it'd be nice if this went in a more standard place like ~/.config/KnobKraftOrm/

**Comment by RadekPilich** [comment]
it's the same mess on Windows - 4 locations 😄

**Comment by christofmuc** [comment]
Agree, this needs to be cleaned up. Reason is as I stated above the JUCE implementation of "well known directories", but I also screwed up somewhere in the middle of using too many of them. On Linux, I wouldn't even know if the distributions agree where it should go, on Windows we should probably stop using the Documents tree...

I am currently traveling, won't find time for KnobKraft before next month I guess.

**Comment by firstnevyn** [comment]
Just for reference:

https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s08.html#homeReferences 

the closest thing to the concept of a 'commonDocumentsDirectory,'  on linux would be something like:

/usr/share/data/KnobKraft/Adaptations

But.. having the app write there would be somewhat unexpected unless it was installed as an integrated package.
